src/pages/Main/Main.tsx

```diff
@@ -1,31 +1,40 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Link } from 'react-router-dom';
 
 import Layout from 'components/Layout';
 import Image from 'components/Image';
-
 import mainImage from 'images/main.jpeg';
+import getTokenInfo from 'utils/getTokenInfo';
+import getAuthData from 'utils/getAuthData';
 
-const Main = (): JSX.Element => (
-  <Layout className="container mx-auto p-10">
-    <div className="mr-20 w-7/12 rounded overflow-hidden">
-      <Image className="min-w-full" src={mainImage} />
-    </div>
-    <div>
-      <div className="text-5xl mb-6">Играйте в Battle Dice</div>
-      <div className="max-w-lg text-gray-500 mb-14">
-        Лучшее место для игры с друзьями или случайными людьми. Чтобы начать
-        игру вам нужно просто зарегистрироваться. Никаких платежей и подписок -
-        сразу можете начать игру!
-      </div>
-      <Link
-        to="/game"
-        className="uppercase font-medium py-2 px-4 bg-blue-700 text-white rounded my-1 md:mr-5 md:my-0 hover:bg-blue-800 min-w-full block text-center"
-      >
-        Играть
-      </Link>
-    </div>
-  </Layout>
-);
+export default function Main() {
+  useEffect(() => {
+    async function oauthStart() {
+      await getTokenInfo();
+      await getAuthData();
+    }
+    oauthStart();
```

Comment:
❗️
По имени опционально, но желательно:
1. `start` в названии подразумевает, что ещё где-то есть `stop`/`finish`/`exit`/`pending`. А их по факту нет
2. В таком варианте название функции звучит как "Начало OAuth", а не "Начать OAuth". А оно должно быть глаголом
```suggestion
    async function processOAuth() {
      await getTokenInfo();
      await getAuthData();
    }
    
    processOAuth();
```